### PR TITLE
Refactoring code - almost no logic changes

### PR DESCRIPTION
**Pull request by metalice**
Hey :)

Saw this repo a couple of hours a go and liked the idea, made some changes to the code and reactors to be more FE , more react way

### CODE REVIEW

File: src/utils/utils.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  // Ensure it's within 0-7
+  degrees = (degrees + 8) % 8;
+  return DIRECTIONS?.[degrees];
+};
+
+export const getFetchURL = (location: LatLngLocation): string => {
+  return `https://he.wikipedia.org/w/api.php?action=query&list=geosearch&gscoord=${location?.lat}|${location?.lng}&gsradius=2000&gslimit=50&format=json&gsprop=type|name&inprop=url&prop=info&origin=*`;
+};
+
+export const addImageToResult = (results: Geosearch[], y: any) => {
```
Comment (noam-honig) [inline src/utils/utils.ts]: it's not just the image, it's also the description :)

File: src/components/ResultEntry/ResultEntry.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import { Geosearch, LatLngLocation } from "../../utils/types";
+import { direction, getGoogleMapLink, getResultLink } from "../../utils/utils";
+
+import "./result-entry.scss";
+
+type ResultEntryProps = { result: Geosearch; location: LatLngLocation | undefined };
+
+const ResultEntry: FC<ResultEntryProps> = ({ result, location }) => {
+  return (
+    <tr className="ResultEntry--main">
```
Comment (noam-honig) [inline src/components/ResultEntry/ResultEntry.tsx:11]: I'm not familiar with these class names from the scss file - do you get autocomplete for these? or is just some a code standard that I'm not familiar with - would love more info

File: package.json [hunk 3]
```diff
@@ -16,7 +16,8 @@
     "@types/react": "^18.0.27",
     "@types/react-dom": "^18.0.10",
     "@vitejs/plugin-react": "^3.1.0",
+    "sass": "^1.59.3",
```
Comment (noam-honig) [inline package.json:19]: Does vite automatically recognizes this, or does it require configuring a plugin for vite?

**Review by noam-honig — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (metalice) [reply package.json:19]: auto :)

File: src/components/ResultEntry/ResultEntry.tsx (hunk 2 above)
Comment (metalice) [reply src/components/ResultEntry/ResultEntry.tsx:11]: The names are set the B.E.M way, in the scss the & and such are part of scss lang, its can be more sophisticated and complicated, but I kept it simple.

File: src/utils/utils.ts (hunk 1 above)
Comment (metalice) [reply src/utils/utils.ts]: ok so lets change it addDataToResult

**Comment by noam-honig** [comment]
יש פספוס קטן - גודל הפונט של הכותרת קטן - אני אתקן
image

image

**Comment by noam-honig** [comment]
@metalice 
Ok - I need your help, please :)

The font size doesn't work anymore and I'm not sure why - in chrome, it says:
image

Although it seems ok - even if I change it to pixel values, it doesn't take that.

If I place it in a style, it does work - but I want it to work the right way. Any chance you'll take a look?

In the meantime, I've put the x-large in a style (don't hate me) 
image

**Comment by noam-honig** [comment]
One more thing, the media query that made the images small on mobile also doesn't work anymore

https://github.com/noam-honig/wikiloc/blob/c6d3063516a9c36151d9cd827342f51f46d444ef/src/index.css#L76

**Comment by noam-honig** [comment]
And also the white-space doesn't work with that:
image
image

And the font size small :)

**Pull request merged**